chalice/awsclient.py

```diff
@@ -79,7 +80,7 @@ def create_function(self,
         # type: (...) -> str
         kwargs = {
             'FunctionName': function_name,
-            'Runtime': 'python2.7',
+            'Runtime': DEFAULT_LAMBDA_RUNTIME,
```

Comment:
Should we even have this as a default value?  I think it should always match whatever version of python you're using and perhaps even be a required param.  The deployer will always pass this value anyways.